packages/turf-clusters-dbscan/lib/rbush-export.ts

```diff
@@ -0,0 +1,7 @@
+// Get around problems with moduleResolution node16 and some older libraries.
+// Manifests as "This expression is not callable ... has no call signatures"
+// https://stackoverflow.com/a/74709714
+
+import lib from "rbush";
+
+export const rbush = lib as unknown as typeof lib.default;
```

Comment:
I'm a little surprised to see this being a problem.
https://github.com/mourner/rbush/blob/master/index.js has `export default class RBush`
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/rbush/index.d.ts has `export default class RBush<T>`

Are we running an old rbush types or js package perhaps?